src/motor_hardware.cc

```diff
@@ -794,6 +851,10 @@ void MotorDiagnostics::firmware_options_status(DiagnosticStatusWrapper &stat) {
     } else {
         option_descriptions +=  ", Standard wheels";
     }
+    if (firmware_options & MotorMessage::OPT_WHEEL_DIR_REVERSE) {
+        // Only indicate wheel reversal if that has been set as it is non-standard
+        option_descriptions +=  ", Reverse wheels";
```

Comment:
Can you make it "reverse direction" Otherwise right now we might get "Standard Wheels, Reverse wheels" which is a little confusing.

Reply:
Humm.  Even that too is not enough info.  I see your point.   How about  'Reverse polarity wheels'?
I am trying to say in a short number of words enough to convey this:
'Wheels that when driven by the standard motor controller turn in the reverse way from standard wheels'

Reply:
I'll try 'Reverse polarity wheels'